feat: add external documentation (#275)
okteto/docs: src/content/reference/manifest.mdx

Sets the working directory of your development container.

### external (object, optional)

> A list of external resources to deploy with your development environment. This external resources contains the information for resources that are deployed outside the okteto cluster.

Reviewer: Given that this iteration doesn't deploy anything, do we need to change the documentation?

```suggestion
A list of external resources that are part of your development environment.  Use this section for resources that are deployed outside of the okteto cluster, like Cloud resources or dashboards.
```

```yaml
external:
  db:
    notes: docs/database.md
    endpoints:
    - name: db
      url: https://localhost:3306
  functions:
    notes: docs/lambdas.md
    endpoints:
    - name: data-aggregator
      url: https://fake-id.lambda-url.us-east-1.on.aws
    - name: data-processor
      url: https://fake-id.lambda-url.us-east-2.on.aws
```

#### endpoints ([object], required)

> Endpoints contains the information about how to access the external resource.

Reviewer: ```suggestion
Endpoints contain information on how to access the external resource.
```

##### name (string, required)

> The name to identify the endpoint.

Reviewer: ```suggestion
The name of the endpoint. 
```

##### url (string, optional)

> The url to access the endpoint.

Reviewer: ```suggestion
The url of the endpoint.
```

#### notes (string, optional)

> A path containing a markdown document explaining what the external resource is about.

Reviewer: ```suggestion
A relative path to a mardown file. The contents of the file will be displayed in the dashboard. Use this to describe the resource or share instructions on how to use it.
```

Sets the working directory of your development container.

### external (object, optional)

A list of external resources to deploy with your development environment. This external resources contains the information for resources that are deployed outside the okteto cluster.

```yaml
external:
  db:

>     icon: database

Reviewer 2: Is `icon` already implemented? Or is the intention to prepare docs for upcoming implementation? If the latter, should we consider adding the icon details later on when the implementation is done & ready?

Author: Since only the current status will make it into 1.4 we have to remove icon field from the docs. I think it will be implemented next sprint

    notes: docs/database.md
    endpoints:
    - name: db
      url: https://localhost:3306
  functions:
    icon: function
    notes: docs/lambdas.md
    endpoints:
    - name: data-aggregator
      url: https://fake-id.lambda-url.us-east-1.on.aws
    - name: data-processor
      url: https://fake-id.lambda-url.us-east-2.on.aws
```

#### endpoints ([object], required)

Endpoints contains the information about how to access the external resource.

##### name (string, required)

The name to identify the endpoint.

##### url (string, optional)

The url to access the endpoint.

> #### icon (string, optional)

Reviewer 3: is `icon` already supported? If not, I wouldn't document it yet
